### PR TITLE
GETEXwith EX with big integer reports an error

### LINKED ISSUE #510: Inconsistent `GETEX`: GETEX with big integer should report an error
## Steps to reproduce

Run the commands mentioned in the test on Line 98 in the file https://github.com/AshwinKul28/dice-tests/tree/main/tcltests/unit/expire.tcl#L98.

## Expected output

The expected output when the above set of commands when run on Redis

```
ERR invalid expire time in 'getex' command
```

## Observed output

The observed output when the above set of commands when run on DiceDB

```
bar
```

The steps to run the test cases are mentioned in the README of the [dice-tests repository](https://github.com/AshwinKul28/dice-tests).

## Expectations for resolution

This issue will be considered resolved when the following things are done

1. changes in the [`dice`](https://github.com/dicedb/dice) code to meet the expected behavior
2. Successful run of the tcl test behavior

You can find the tests under the `tests` directory of the [`dice`](https://github.com/dicedb/dice) repository and the steps to run are in the [README file](https://github.com/dicedb/dice). Refer to the following links to set up DiceDB and Redis 7.2.5 locally

- [setup DiceDB locally](https://github.com/dicedb/dice)
- [setup Redis 7.2.5 locally](https://gist.github.com/arpitbbhayani/94aedf279349303ed7394197976b6843)
- [setup DiceDB CLI](https://github.com/dicedb/dice)

### PR DESCRIPTION
**Pull request by Gaurav-Kumar81**
close #510 

Passes TCL test

image

Screenshot from 2024-09-14 01-44-49

Working Example : 
image

### CODE REVIEW

**Comment by AshwinKul28** [comment]
Hi @Gaurav-Kumar81 looks good, can you please add a small unit test for this? Thanks

**Comment by Gaurav-Kumar81** [comment]
@AshwinKul28  where i can add the unit test.

**Comment by Gaurav-Kumar81** [comment]
@AshwinKul28 Please have look on the testcase i have added

**Comment by AshwinKul28** [comment]
Hey @Gaurav-Kumar81 Thanks for the changes. LGTM, left a minor comment.

**Comment by Gaurav-Kumar81** [comment]
@AshwinKul28  I am not able see your comments.

File: internal/eval/eval.go [hunk 1]
```diff
@@ -1872,7 +1872,7 @@ func evalGETEX(args []string, store *dstore.Store) []byte {
 			if err != nil {
 				return diceerrors.NewErrWithMessage(diceerrors.IntOrOutOfRangeErr)
 			}
-			if exDuration <= 0 {
+			if exDuration <= 0 || exDuration >= maxExDuration {
```
[the remark this answers is not in the document]
Comment (Gaurav-Kumar81) [reply internal/eval/eval.go]: @AshwinKul28  i did the changes thanks for you help.

**Review by AshwinKul28 — COMMENTED** [review]
Awesome, LGTM. Thanks a lot for this @Gaurav-Kumar81

**Pull request merged**